phys2bids/io.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
+"""phys2bids interfaces for txt  and acq extension files."""
 
 import logging
 from collections import Counter
```

Comment:
I can't comment at the right line (again), but here's a little thing that might have been overlooked in #347.
Line 11: `bioread` is not a mandatory dependency (nor it should be) - it's only used if acq file have to be processed.
Before, when each interface had its own script, it was not a problem because the acq interface would have been called ONLY if an acq file was going to be run. Now that the interfaces are going to be initialised anyway, if this import is left here, the program will try to import `bioread` even if a txt file is read - and if the user doesn't have it installed (given that it's not a mandatory library), it will raise an error.
Two solutions for this: either you move this import to the first line in `load_acq`, or you `try` this import and raise an error in load_acq if it's missing. I suggest to go for the first solution - makes the code easier.

Reply:
BTW, this will be the same for matlab related libraries.